components/brave_shields/browser/BUILD.gn

```diff
@@ -21,6 +23,11 @@ source_set("browser") {
     "ad_block_service.h",
     "ad_block_service_helper.cc",
     "ad_block_service_helper.h",
+    "ad_block_subscription_service.cc",
```

Comment:
there seems to be inconsistent terminology here which is confusing. Sometimes it is `custom_subscription` and sometimes `ad_block_subscription`. Please use consistent terminology